docs/blogs/links-syphon/syphon.md

```diff
@@ -0,0 +1,151 @@
+management: it slots into your existing lineup of tools.  So a significant
+part of our work on lakeFS is devoted to leveraging lakeFS and these other
+tools to improve these integrations.  Our latest addition is Project Syphon!
+
+<img alt="Soda syphon, by Avi Nahmias, available from https://commons.wikimedia.org/wiki/User:Avin" src="https://upload.wikimedia.org/wikipedia/commons/1/1c/Soda_syphon_IMG_1505C.jpg" width=400/>
+
+Airflow was one of our earliest integrations: Itai published the original
+[Air and Water][airflow-air-and-water-blog-1] 2 years ago.  It's time to
+deepen our integrations!  The latest release of the lakeFS provider for
+Airflow, codename "Project Syphon", adds deep linking and metadata between
+Airflow DAG runs and the lakeFS commits that they create.  It connects the
+two for interactive or programmatic use.
+
+The new Airflow operators add formatted metadata to your commits.  lakeFS
+uses this metadata to link to Airflow.  Project Syphon adds flexibility to
+these keys by defining a format for metadata keys.  You can leverage such
+metadata to add your own operable metadata on any commits you perform from
+_any_ tools (not only from Airflow).
+
+How are you integrating these new features into your workflows?  How can we
+improve them and help you?  Please let us know on [lakeFS
+Slack][lakefs-slack] or by [opening an issue][airflow-provider-new-issue]!
+
+## Using the lakeFS Airflow provider
+
+Links are generated by lakeFS provider versions 0.46.1 and above.  To use
+them, ensure your Airflow `requirements.txt` includes
+
+```py
+airflow-provider-lakefs>=0.46.1
+```
+
+That is pretty much all.  Using LakeFSCommitOperator or LakeFSMergeOperator
+normally in your DAGs will add metadata to your commits.
+
+Any version of lakeFS can serve commit metadata to clients.  The lakeFS GUI
+adds the ability to interpret Airflow provider metadata to link back to
+Airflow starting at version **TBD TBD** 0.100.0 **TBD TBD**.
+
+### Metadata on lakeFS commits
+
+When examining a commit in lakeFS, the UI shows metadata and a link "Open
+Airflow UI" to the DAG run in the UI.
+
+![lakeFS commit UI with "Open Airflow UI" button][lakefs-commit-ui-open-button]
+
+The button "Open Airflow UI" links to the generating DAG run on Airflow.
+Metadata from the DAG run on Airflow is added to the commit, in keys
+starting "::lakefs::".
+
+### Link on Airflow
+
+When examining the task of a LakeFSCommitOperator or a LakeFSMergeOperator
+of a workflow on Airflow, the UI shows a "lakeFS" button.
+
+![Airflow DAG UI with "lakeFS" button][airflow-task-ui-open-button]
+
+This button links to the generated commit on lakeFS.
+
+## How it works
+
+Knowing how all this works can help you perform additional tasks by
+leveraging these features.  Part of the tooling approach of Airflow and
+lakeFS is to provide simple interfaces that can readily be used to integrate
+and reliably extract data.  Syphon is built by combining tools:
```

Comment:
I like the project code name, but referring to it throughout and not the integration makes me feel like there's an additional bit of software to add to my stack here, not just a newer version of what I already have/am considering installing. Consider instead referring to "the latest version of the airflow operator" / "now with the airflow operator" etc. Keep the project name in the article intro tho, it adds nice personality to the article

Reply:
Sounds like a plan!